internal/patroni/rbac.go

```diff
@@ -12,25 +12,25 @@ import (
 )
 
 // "list", "patch", and "watch" are required. Include "get" for good measure.
-// +kubebuilder:rbac:namespace=patroni,groups="",resources="pods",verbs={get}
-// +kubebuilder:rbac:namespace=patroni,groups="",resources="pods",verbs={list,watch}
-// +kubebuilder:rbac:namespace=patroni,groups="",resources="pods",verbs={patch}
```

Comment:
:thinking: What was `namespace=patroni` doing/signifying here?

Reply:
IIRC, the `namespace` part of the kubebuilder RBAC marker is a way to tell kubebuilder whether we want a ClusterRole or a Role specifically for a namespace:
```
If not set, the Rule belongs to the generated ClusterRole. If set, the Rule belongs to a Role, whose namespace is specified by this field.
```
[Citation](https://book.kubebuilder.io/reference/markers/rbac)

We were setting the namespace here (from the beginning of this file's existence, afaict) as a way to remind ourselves that these permissions were required by patroni; and then in our RBAC generation, we were (essentially) stripping that namespace out so that these permissions would just be part of the ClusterRole/Role that you need to run PGO. 

Now with some updates to controller-gen, we don't need to do that bit of processing; and since this rbac is specifically in the patroni directory, that's enough to tell us what these permissions are for, so we can drop the `namespace` field.

Reply:
:+1: That makes sense, thanks for the explanation.